scriptmodules/emulators/jzintv.sh

```diff
@@ -18,7 +18,7 @@ rp_module_section="opt"
 rp_module_flags="sdl2"
 
 function depends_jzintv() {
-    getDepends libsdl2-dev libreadline-dev
+    getDepends libsdl2-dev libreadline-dev flip
```

Comment:
BTW we don't use flip anywhere else. dos2unix is used though so that would be a preference. But you can probably just use sed though to change the files that are needed (or include the patch with CRLF line endings and avoid having to change the source).